smooth/components/component_storage_h2.py

```diff
@@ -18,15 +18,17 @@ def __init__(self, params):
         # Min. and max. pressure [bar].
         self.p_min = 0
         self.p_max = 450
-        # Storage capacity capacity at p_max (assuming all the can be used,
+        # Storage capacity at p_max (assuming all the can be used,
         # p_min is not included here) [kg].
```

Comment:
now minimal capacity according to p_min is included, so not all of it can be used

Reply:
I will correct the comment, push it to the branch and merge it